<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BusView</class>
 <widget class="QMainWindow" name="BusView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Memory Bus Logger</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/appResources.qrc">
    <normaloff>:/resources/Logo.png</normaloff>:/resources/Logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTableView" name="log">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QFrame" name="toolbox">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="midLineWidth">
        <number>2</number>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="Filters">
          <property name="title">
           <string>Filters</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QGroupBox" name="formForm">
             <property name="title">
              <string>Read from</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="cPULabel_4">
                <property name="text">
                 <string>CPU</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QCheckBox" name="fromCPU">
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="aPULabel_4">
                <property name="text">
                 <string>APU</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QCheckBox" name="fromAPU">
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="pPULabel_4">
                <property name="text">
                 <string>PPU</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QCheckBox" name="fromPPU">
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="rOMLabel_3">
                <property name="text">
                 <string>ROM</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QCheckBox" name="fromROM">
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="rAMLabel_3">
                <property name="text">
                 <string>WRAM</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QCheckBox" name="fromWRAM">
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="sRAMLabel_3">
                <property name="text">
                 <string>SRAM</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QCheckBox" name="fromSRAM">
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="vRAMLabel_3">
                <property name="text">
                 <string>VRAM</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QCheckBox" name="fromVRAM">
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="oAMRAMLabel_3">
                <property name="text">
                 <string>OAM RAM</string>
                </property>
               </widget>
              </item>
              <item row="7" column="1">
               <widget class="QCheckBox" name="fromOAM">
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="cGRAMLabel_3">
                <property name="text">
                 <string>CG RAM</string>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QCheckBox" name="fromCG">
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="9" column="0" colspan="2">
               <widget class="QPushButton" name="fromToggle">
                <property name="text">
                 <string>Toggle all</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="toFrom">
             <property name="title">
              <string>Write to</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QLabel" name="cPULabel_3">
                <property name="text">
                 <string>CPU</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QCheckBox" name="toCPU">
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="aPULabel_3">
                <property name="text">
                 <string>APU</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QCheckBox" name="toAPU">
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="pPULabel_3">
                <property name="text">
                 <string>PPU</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QCheckBox" name="toPPU">
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="rOMLabel_2">
                <property name="text">
                 <string>ROM</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QCheckBox" name="toROM">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
                <property name="tristate">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="rAMLabel_2">
                <property name="text">
                 <string>WRAM</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QCheckBox" name="toWRAM">
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="sRAMLabel_2">
                <property name="text">
                 <string>SRAM</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QCheckBox" name="toSRAM">
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="vRAMLabel_2">
                <property name="text">
                 <string>VRAM</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QCheckBox" name="toVRAM">
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="oAMRAMLabel_2">
                <property name="text">
                 <string>OAM RAM</string>
                </property>
               </widget>
              </item>
              <item row="7" column="1">
               <widget class="QCheckBox" name="toOAM">
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="cGRAMLabel_2">
                <property name="text">
                 <string>CG RAM</string>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QCheckBox" name="toCG">
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="9" column="0" colspan="2">
               <widget class="QPushButton" name="toToggle">
                <property name="text">
                 <string>Toggle all</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="clear">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="2" column="0">
            <widget class="QPushButton" name="clearBtn">
             <property name="text">
              <string>Clear</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QWidget" name="widget_2" native="true"/>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="autoscroll">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Autoscroll</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/appResources.qrc"/>
 </resources>
 <connections/>
</ui>
